### PR TITLE
Misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3`

### CODE REVIEW

File: src/gmic.cpp [hunk 1]
```diff
@@ -6548,7 +6548,7 @@ gmic& gmic::_run(const CImgList<char>& commands_line, unsigned int& position,
             sx = cimg::round(sx);
             sy = cimg::round(sy);
             sz = cimg::round(sz);
-            print(images,0,"Dilate image%s with %gx%gx%g kenrel and neumann boundary conditions.",
+            print(images,0,"Dilate image%s with %gx%gx%g kernel and neumann boundary conditions.",
```
Comment (luzpaz) [inline src/gmic.cpp]: please review

File: html/tutorials/anisotropic_smoothing_zonder/index.shtml [hunk 2]
```diff
@@ -34,9 +34,9 @@ gmic clean_disc.png -noise 30 -n 0,255 -o disc.png</I></FONT><BR>
...
 <BR>
-Once the gradient computations are done, it is obtained what is called a <I>structure tensor</I>, that defines to the filter where it should act most (far from the structure borders). In practice, this <I>structure tensor </I>undergoes a second blurring in order to reduce the abruptness of the structure definition, thus increasing the size of the zones where the filter should have a intermediate behavior. The parameter regulating this second blurring is called the <B>tensor smoothness</B>. It can (for instance) diminish artefacts that may appear close to structure borders.<BR>
+Once the gradient computations are done, it is obtained what is called a <I>structure tensor</I>, that defines to the filter where it should act most (far from the structure borders). In practice, this <I>structure tensor </I>undergoes a second blurring in order to reduce the abruptness of the structure definition, thus increasing the size of the zones where the filter should have a intermediate behavior. The parameter regulating this second blurring is called the <B>tensor smoothness</B>. It can (for instance) diminish artifacts that may appear close to structure borders.<BR>
```
Comment (luzpaz) [inline html/tutorials/anisotropic_smoothing_zonder/index.shtml]: github doesn't do a good job of differentiating the change here but I s/artefacts/artifacts/

File: resources/gmicol.gmic [hunk 3]
```diff
@@ -81,7 +81,7 @@
 
 #@web Ascii art : fx_asciiart, fx_asciiart(0)
 #@web : Charset = choice(5,"Custom","Binary digits","Digits","Lowercase letters","Uppercase letters","Ascii","Card suits","Math symbols")
-#@web : Custom dictionnary = text{" .oO0"}
+#@web : Custom dictionary = text{" .oO0"}
```
Comment (luzpaz) [inline resources/gmicol.gmic]: please review this as well and the commit below

**Comment by dtschump** [comment]
Very useful, thank you !!

**Pull request merged**